dc-infrastructure.tf

```diff
@@ -118,6 +136,7 @@ module "jira" {
   namespace               = module.base-infrastructure.namespace
   vpc                     = module.base-infrastructure.vpc
   eks                     = module.base-infrastructure.eks
+  rds                     = module.database[index(var.products, "jira")]
   ingress                 = module.base-infrastructure.ingress
   db_major_engine_version = var.jira_db_major_engine_version
```

Comment:
Since rds module is no longer in product module, we don't really need `db_major_engine_version` here anymore.